README.md

```diff
@@ -2,31 +2,200 @@
+  - A table holds information about objects of a similar type.
+  - Each column within a table has a unique name.
+  - All values within a column have the same type and are atomic.
+  - Each row within a table represents a unique object and must have a unique identifier, called a **primary key**.
+  - Rows among multiple tables can be related using **foreign keys**, in which one or more columns in
+    one table store the values of the primary key of another table.
+- A **query language** is a specialized programming language designed for database systems.
+  - **Structured Query Language** or **SQL** is a query language designed for relational databases.
+- **PostgreSQL** is a popular open source RDBMS.  Other RDMBS products include Oracle, MySQL, SQLite, and SQL Server.
+
+## Database Structure
+
+Relational databases store data in table that consists of
+columns and rows.  Each row contains data about a unique
+entity or object, such as a person, product, or event.
+Each column stores a single atomic attribute
+such as a person's date of birth, a product price, or a meeting date.
+
+For example, assume we want a database to store information about
+pets and their owners.  We will need one table to store data about
+owners, and a separate table to store data about pets.
+Our database model should specify the attributes
+to store for each owner and each pet, along with the cardinality of the
+relationship between owner and pet. We'll assume each pet has one owner,
+while an owner may own many pets.   Thus, our `owner` and `pet` tables
+need columns to store the following data:
+
+- **owner**: unique identifier, first name, last name, and phone number.
+- **pet**: unique identifier, name, species (cat, dog, etc), breed, age, and owner's unique identifier.
+
+An **entity-relationship model** of `owner` and `pet` is shown below.  The relationship connecting
+pet and owner indicates each pet has exactly one owner ( indicated by single line | ),
+while an owner may have many pets ( indicated by three forked lines, also called "crows feet" ).
+This type of relationship is called "one-to-many".
+
+![pet er diagram](https://curriculum-content.s3.amazonaws.com/6036/database-basics/pet_erd.png)
+
+
+Example database tables containing data for 3 owners and 9 pets could look like the following:
+
+| owner table                                                                                      | pet  table                                                                                   |
+|--------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------|
+| ![owner table](https://curriculum-content.s3.amazonaws.com/6036/database-basics/owner_table.png) | ![pet table](https://curriculum-content.s3.amazonaws.com/6036/database-basics/pet_table.png) |
```

Comment:
Lil Bub! I love Lil Bub! He was so cute. RIP. I love this example - I usually use a pet example too when describing databases.

Reply:
:-)